examples/amgx/ex1.cpp

```diff
@@ -205,10 +205,14 @@ int main(int argc, char *argv[])
 
       if (amgx_solver)
       {
+         amgx.SetConvergenceCheck(true);
          amgx.Mult(B,X);
       }
       else
       {
+         //Omit convergence check at the AmgX level
+         //when using as a preconditioner
```

Comment:
Styling comment: in the mfem source we leave a space after `//` and align on 80 characters 😁